src/main/java/nl/tudelft/ti2206/group9/audio/SoundtrackPlayer.java

```diff
@@ -0,0 +1,204 @@
+        } catch (MediaException me) {
+            OBSERVABLE.notify(Category.ERROR, Error.MEDIAEXCEPTION,
+                    "SoundtrackPlayer.stop()", me.getMessage());
+        } catch (NullPointerException ne) { // NOPMD
+            // This try-catch block is just here for testing.
+            // The stop method can result in a NullPointer (according to
+            // JUnit), because JUnit can't really play audio neither can Travis.
+            OBSERVABLE.notify(GameObserver.Category.ERROR,
+                    GameObserver.Error.NULLPOINTEREXCEPTION,
+                    "SoundtrackPlayer.stop()", ne.getMessage());
+        }
+    }
+
+    /**
+     * Pauses the SoundtrackPlayer,
+     * if the sound is enabled.
+     */
+    public final void pause() {
+        try {
+            if (State.isSoundtrackEnabled()) {
+                if (mediaPlayer == null) {
+                    initializeAudio(path);
+                }
+                mediaPlayer.pause();
+            }
+        } catch (MediaException me) {
+            OBSERVABLE.notify(Category.ERROR, Error.MEDIAEXCEPTION,
+                    "SoundtrackPlayer.pause()", me.getMessage());
+        } catch (NullPointerException ne) { // NOPMD
+            // This try-catch block is just here for testing.
+            // The pause method can result in a NullPointer (according to
+            // JUnit), because JUnit can't really play audio neither can Travis.
+            OBSERVABLE.notify(GameObserver.Category.ERROR,
+                    GameObserver.Error.NULLPOINTEREXCEPTION,
+                    "SoundtrackPlayer.play()", ne.getMessage());
+        }
+    }
+
+    @Override
+    public final boolean isRunning() {
+        if (mediaPlayer == null) {
+            return false;
+        } else {
+            return mediaPlayer.getStatus().equals("PLAYING");
+        }
+    }
+
+    /**
+     * Increases the speed of the soundtrack played by the SoundtrackPlayer.
+     * @param rate rate to be set to.
+     * It is recommended to keep this rate between 0.125 and 8.0.
+     */
+    public final void setSpeed(final double rate) {
+        if (State.isSoundtrackEnabled() & rate >= LOWER & rate <= UPPER) {
+            mediaPlayer.setRate(rate);
+        }
+    }
+
+    /**
+     * Gets the current speed to which the SoundtrackPlayer is set.
+     * (For testing purposes.)
+     * @return double rate current rate of the SoundtrackPlayer.
+     */
+    public final double getSpeed() {
+        if (State.isSoundtrackEnabled()) {
+            return mediaPlayer.getRate();
+        }
+        return 1.0;
+    }
+
+    /**
+     * Resets the speed of the soundtrack played by the SoundtrackPlayer.
+     */
+    public final void resetSpeed() {
+        if (State.isSoundtrackEnabled()) {
+            mediaPlayer.setRate(1.0);
+        }
+    }
+
+    /**
+     * Loops the SoundtrackPlayer.
+     */
+    private void loopAudio() {
+        try {
+        mediaPlayer.setCycleCount(INDEFINITE);
```

Comment:
Incorrect indentation on this line. Try Ctrl+A Ctrl+I (that works in my Eclipse), it fixes all indentation for this file automatically :) Provided that your indent settings are correct, of course :P

<review, layout